### PR TITLE
[NANO]: Remove useless appendages from lnwallet/channel.go

### PR DESCRIPTION
**Pull request by ProofOfKeags**
## Change Description
gopls informed me that these parameters are not used and when I remove them, they are indeed not necessary.

## Steps to Test
make unit pkg=lnwallet

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [ ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ] Any new lncli commands have appropriate tags in the comments for the rpc in the proto file.
- [ ] [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

File: lnwallet/channel.go [hunk 1]
```diff
@@ -5198,7 +5198,6 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
 	localChainTail := lc.localCommitChain.tail().height
 
 	source := lc.ShortChanID()
-	chanID := lnwire.NewChanIDFromOutPoint(lc.channelState.FundingOutpoint)
```
Comment (yyforyongyu) [inline lnwallet/channel.go:5212]: looks like it's still been used below.

**Review by yyforyongyu — CHANGES_REQUESTED** [review]
CI failed.

**Comment by ProofOfKeags** [comment]
I got trigger happy with the commit reordering turns out the last commit needed to be part of a different series

File: lnwallet/channel.go [hunk 2]
```diff
@@ -566,7 +566,7 @@ func (c *commitment) toDiskCommit(
 // restore commitment state written to disk back into memory once we need to
 // restart a channel session.
 func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
-	commitHeight uint64, htlc *channeldb.HTLC, localCommitKeys,
+	htlc *channeldb.HTLC, localCommitKeys *CommitmentKeyRing,
 	remoteCommitKeys *CommitmentKeyRing, whoseCommit lntypes.ChannelParty,
```
Comment (carlaKC) [inline lnwallet/channel.go:570]: Unnecessary `*CommitmentKeyRing` after `localCommitKeys`, same for `extractPayDescs`

**Review by carlaKC — APPROVED** [review]
935e7f1bee3a5d61bf80d5db1cd43bad48ecaea0 lgtm

**Review by yyforyongyu — APPROVED** [review]
LGTM🦾

File: lnwallet/channel.go (hunk 2 above)
Comment (carlaKC) [reply lnwallet/channel.go:570]: feelz

omsac

**Comment by ProofOfKeags** [comment]
Failure is a flake. Send it.

**Pull request merged**